benchmark_runner/common/oc/oc.py

```diff
@@ -715,6 +717,22 @@ def collect_events(self):
         self.run(f"{self.__cli} get events -A > '{output_filename}' ")
         return output_filename
 
+    def get_kube_api_server(self):
+        """
+        This method returns server url from config file
+        :return: server url
+        """
+        try:
+            with open(self.__kubeconfig_path, "r") as f:
```

Comment:
The kubeconfig file is a YAML file.  Rather than reading the file in this way, load the YAML file appropriately and extract the correct entry.

Reply:
@RobertKrawitz  Thx for feedback,
I have updated it using yaml package.